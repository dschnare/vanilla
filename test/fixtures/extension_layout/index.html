<v:extends file="./layout.html" />

<v:append name="meta">
  <meta charset="utf-8" />
</v:append>

<v:replace name="body">
  <p>This is the body!</p>
</v:replace>

<v:block name="any">
  this should be ignored.
</v:block>