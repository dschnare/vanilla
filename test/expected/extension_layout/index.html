<html>
  <head>
    <title>Index</title><meta charset="utf-8" />
  </head>
  <body>
    
    <p>This is the body!</p>
    
  </body>
</html>